README.md

```diff
@@ -32,24 +32,6 @@ The generator is tested via its output when run against `openapi.yml`, viz., the
 
 Beyond what is necessary to test the generator, the Cocina model classes are not tested, i.e., they are assumed to be as specified in `openapi.yml`.
 
-## Generate Documentation
-
-```
-gem install prmd
-cd docs
-
-# Combine into a single schema
-prmd combine --meta meta.json  maps/ > schema.json
-
-# Check it’s all good
-prmd verify schema.json
-
-# Build docs
-prmd doc schema.json > schema.md
-```
-
-Then check in the resulting changes to `docs/schema.json` and `docs/schema.md`
-
 ## Using this gem
```

Comment:
Can we add a link to the openapi docs?

Reply:
It's already in the top of the page: 
<img width="797" alt="Screen Shot 2020-04-01 at 12 51 40 PM" src="https://user-images.githubusercontent.com/92044/78169784-915bbe00-7417-11ea-989d-2e0ca336aa09.png">